grails-app/domain/org/pih/warehouse/shipping/ShipmentItem.groovy

```diff
@@ -93,15 +93,18 @@ class ShipmentItem implements Comparable, Serializable {
     }
 
     String getOrderItemId() {
-        return orderItems?.collect { OrderItem orderItem -> orderItem.id }?.unique()?.first()
+        def orderItemId = orderItems?.collect { OrderItem orderItem -> orderItem.id }?.unique()
+        return orderItemId ? orderItemId.first() : ''
```

Comment:
Why the empty string? I was expecting the default to be null. Also, while we're at it, let's change the plurality on the local variable used in each of these methods i.e. `orderItemId` to `orderItemIds`